pgdb/api.py

```diff
@@ -795,6 +795,16 @@ def create(self, request, corpus_pk=None, *args, **kwargs):
                     str(r) + 'The subset cannot be empty.',
                     status=status.HTTP_400_BAD_REQUEST)
 
+        #Stress pattern validation
+        elif request.data['enrichment_type'] in ['patterned_stress']:
+            #q = models.Enrichment.objects.filter(corpus=corpus).all()
+            #for r in q:
+            r = request.data
+            if 'word_property' not in r or r['word_property'] is None:
+                return Response(
+                    str(r) + 'There must be a word property.',

```

Comment:
We probably don't want to return a copy of the request data along with the response.